samples/Client/DaprClient/Readme.md

```diff
@@ -22,3 +22,80 @@ The client sample shows how to make Dapr calls to publish events, save state, ge
   Got State: my data
   Deleted State!
  ```
+
+## Invoking Services
+This solution contains a sample REST *RoutingSample* service, which implements a simple banking application in ASP.NET core.
+The service provides following operations:
+- balance
+- withdraw
+- deposite
+
+The service is a typical HTTP/S REST service. However, by using **dapr**, another application can invoke an operation on this service by using not only REST.
+Following sample demonstrates how to use the gRPC with Dapr .NET SDK to invoke REST service operations.
+
+Operation *balance* is a GET-operation mapped as (see startup.cs in the *RoutingService*):
+ ```c#
+ endpoints.MapGet("{id}", Balance);
+ ```
+
+It can be invoked via HTTTP/S by using following url:
+ ```
+curl -X GET http://127.0.0.1:5000/17
+ ```
+
+If you are developing the .NET client and want to invoke this operation via HTTP/S, you can also use *HttpClient* API.
+The **dapr** supports out-of-the-box invokation of the service operation via gRPC.
+To demonstrate this, take a look on the method *InvokeBalanceServiceOperationAsync*.
+Services are typically invoked with the following code:
+
+```c#
+var metaData = new Dictionary<string, string>();
+metaData.Add("http.verb", "GET");
+
+var res = await client.InvokeMethodAsync<object serviceName, "17", metadata);
+```
+In this example, the HttpGet method is onvoked on the service with name *serviceName*. This argument specifies the name of the service, which was used when in conjunction with *dapr run –app-id serviceName*.The *route* argument specifies the route of the service endpoint.
+For example, if your service is the Rest service, the route is typically dened by attribute *HttpPost(“route”)* or *HttpGet(“route”)*. Last argument is called metadata and it specifies GET operation in tis example.
+
+
+Second service operation *withdraw* can be invoked by HTTP/S POST request, but also triggered as a *cloud event* by publishing the event to the topic with name 'withdraw'.
+To enable this, the operation is mapped as:
+ ```c#
+endpoints.MapPost("withdraw", Withdraw).WithTopic("withdraw");
+ ```
+and it can also be invoked (triggered) by following url:
+
+ ``` 
+curl -X POST http://127.0.0.1:5000/withdraw -H "Content-Type: application/json" -d "{ \"id\": \"17\", \"amount\": 1 }"
```

Comment:
the curl commands end up being little different due to escaping on windows and linux. COudl you add commands for both as done here https://github.com/dapr/dotnet-sdk/tree/master/samples/AspNetCore/RoutingSample#examples

Reply:
I added curl spinets for Linux and MacOS